site/tutorials/snippets/18/install/cluster.md

```diff
@@ -0,0 +1,18 @@
+## Bring your own Kubernetes cluster
+
+Keptn can be installed on a variety of Kubernetes distributions. Please find a full compatibility matrix for supported Kubernetes versions [here](https://keptn.sh/docs/install/k8s-support/).
+
+Positive
+:  For the sizing of the Kubernetes cluster we recommend a cluster with at least 8vCPUs and 30 GB of memory.
+Detailed sizing recommendations for different platforms can be found in the respective [setup tutorials](../../?cat=installation). 
+
+Please find tutorials [how to set up your cluster here](../../?cat=installation). For the best tutorial experience, please follow the sizing recommendations given in the tutorials.
```

Comment:
Are these tutorials going to survive?  We should have full installation instructions in https://keptn.sh/docs/install/ .  On that page, if you scroll down, you'll find a list of steps that may be required with links for details.  We are still working on this section -- if you see information that is in the tutorials but not in this section, please file an issue or let me know.